src/utils.js

```diff
@@ -34,3 +34,13 @@ export function getCommentForIndividualArticle(article_id) {
             throw err;
         });
 }
+
+export function updateArticleVote(article_id,inc_votes){
+    return api
+    .patch(`articles/${article_id}`,{inc_votes})
+    .then(response => response.data)
```

Comment:
This catch block shouldn't be here. We should only have 1 catch block in a promise chain, at the end (the file in which updateArticleVote was invoked)